### ISSUE TITLE
Broken indentation in raw messages

### ISSUE THREAD

**Issue by amacneil** [foxglove/studio#3846]
**Description**
It appears that when displaying an array with zero items, we lose the correct indentation (see `points` and `colors`):

<img width="637" alt="image" src="https://user-images.githubusercontent.com/637671/179273434-b2c6759f-674f-46e4-a82c-4af227dcaa83.png">

- Version: main
- OS: web / macos

**Comment by amacneil** [comment]
Here is a node playground script you can use to generate a broken marker with the sample dataset:

```ts
import { Input, Message } from "./types";
import { buildRosMarker } from "./markers";

export const inputs = ["/imu"];
export const output = "/broken";

export default function node(
  event: Input<"/imu">,
  globalVars: unknown
): Message<"visualization_msgs/Marker"> {
  return buildRosMarker({});
}
```

**Comment by 2metres** [comment]
Closed by #3841
